tests/config/test_omegaconf_config.py

```diff
@@ -437,7 +438,8 @@ def test_overlapping_patterns(self, tmp_path, mocker):
 
         mocked_load = mocker.patch("omegaconf.OmegaConf.load")
         expected_path = (tmp_path / "dev" / "user1" / "catalog2.yml").resolve()
-        assert mocked_load.called_once_with(expected_path)
+
+        mocked_load.assert_called_once_with(expected_path)
```

Comment:
See https://github.com/python/cpython/issues/100690

This assert was trivially passing, the test was not doing anything (and it's broken at least in 3.11 and 3.12)